maro/cli/process/deployment/process_start_schedule.yml

```diff
@@ -0,0 +1,16 @@
+mode: process
```

Comment:
As we talked before, mode should be one of them:
                "k8s/aks",
                "grass/on-premises",
                "grass/local"

Reply:
There are two mode for local, one is grass/local work with container, another is process work with multiprocess. This is process mode.